NaToxAq/NA000256.txt

```diff
@@ -4,12 +4,11 @@ DATE: 2019.02.28
 AUTHORS: Tobias Schulze, Jawameer Hama, Hubert Schupke, Martin Krauss, Helmholtz Centre for Environmental Research GmbH - UFZ, Leipzig, Germany and University of Copenhagen (UCPH), Denmark
 LICENSE: CC BY
 COPYRIGHT: Copyright (C) 2019
-PUBLICATION: 
 COMMENT: CONFIDENCE Reference Standard (Level 1)
 COMMENT: INTERNAL_ID 14
 CH$NAME: Gramine
 CH$NAME: Gramine
```

Comment:
Repeated name?

Reply:
Thats a good spot. I will put this on the wish list for the validator.